cmake/Zstd.cmake

```diff
@@ -45,8 +45,8 @@ endif()
 set(ZSTD_FOUND "YES" CACHE STRING "" FORCE)
 set(ZSTD_INCLUDE_DIRS "${zstd_SOURCE_DIR}/lib" CACHE STRING "" FORCE)
 if (WIN32)
-    set(ZSTD_LIBRARIES "${zstd_BINARY_DIR}/lib/${CMAKE_BUILD_TYPE}/zstd_static.lib" CACHE STRING "" FORCE)
-    set(ZSTD_LIBRARY "${zstd_BINARY_DIR}/lib/${CMAKE_BUILD_TYPE}/zstd_static.lib" CACHE STRING "" FORCE)
+    set(ZSTD_LIBRARIES "${zstd_BINARY_DIR}/lib/zstd_static.lib" CACHE STRING "" FORCE)
+    set(ZSTD_LIBRARY "${zstd_BINARY_DIR}/lib/zstd_static.lib" CACHE STRING "" FORCE)
```

Comment:
This looks like a change that may be different between ninja and msbuild, and should be tested with both.